client.go

```diff
@@ -0,0 +1,259 @@
+package redis
+
+import (
+	"time"
+
+	"github.com/go-redis/redis/v8"
+	"go.k6.io/k6/js/modules"
+)
+
+// Client represents the Client constructor (i.e. `new redis.Client()`) and
+// returns a new Redis client object.
+type Client struct {
+	vu          modules.VU
+	redisClient *redis.Client
+}
+
+// Set the given key with the given value.
+func (c *Client) Set(key string, value interface{}) {
+	c.redisClient.Set(c.vu.Context(), key, value, 0)
+}
+
+// Get returns the value for the given key.
+func (c *Client) Get(key string) (string, error) {
+	return c.redisClient.Get(c.vu.Context(), key).Result()
+}
+
+// Del removes the specified keys. A key is ignored if it does not exist
+func (c *Client) Del(keys ...string) (int64, error) {
+	return c.redisClient.Del(c.vu.Context(), keys...).Result()
+}
+
+// Exists returns the number of key arguments that exist.
+// Note that if the same existing key is mentioned in the argument
+// multiple times, it will be counted multiple times.
+func (c *Client) Exists(keys ...string) (int64, error) {
+	return c.redisClient.Exists(c.vu.Context(), keys...).Result()
+}
+
+// Incr increments the number stored at `key` by one. If the key does
+// not exist, it is set to zero before performing the operation. An
+// error is returned if the key contains a value of the wrong type, or
+// contains a string that cannot be represented as an integer.
+func (c *Client) Incr(key string) (int64, error) {
+	newValue, err := c.redisClient.Incr(c.vu.Context(), key).Result()
```

Comment:
(here and in similar places) Just wondering if we need `newValue` :thinking: What if we just `return c.redisClient.Incr(c.vu.Context(), key).Result()`?

Reply:
You're right, that's a leftover, will address it 👍🏻